### ISSUE TITLE
Server should log user name in each action

### ISSUE THREAD

**Issue by riccardomurri** [TissueMAPS/TmServer#4]
Server logs (as currently stored in `uwsgi.log`) contain no indication of the user performing an action.  On a multi-user system this makes it very hard to follow the flow of events, as actions taken in response to what a user does are intermixed with other users' doings.

Simply logging the user name in the log message would make it easy to reconstruct the flow for a single user.  If possible, I would add also the session ID to the log line: this would make it possible to distinguish actions that a user performs in different browser windows/tabs.

**Comment by HackerMD** [comment]
I agree!

The current user is available via the variable `current_identity`:

```python
from flask_jwt import current_identity
```

We should evaluate whether it would work to directly add this information to each logging message by customizing the logging [Formatter](https://docs.python.org/2/library/logging.html#logging.Formatter).

**Comment by riccardomurri** [comment]
This issue was moved to TissueMAPS/TissueMAPS#120
